Emby.XmlTv/Emby.XmlTv/Classes/XmlTvReader.cs

```diff
@@ -495,16 +495,33 @@ public void ProcessEpisodeNum(XmlReader reader, XmlTvProgram result)
                     ParseMovieDbSystem(reader, result);
                     break;
                 case "SxxExx":
-                    // TODO
-                    // <episode-num system="SxxExx">S03E12</episode-num>
-                    reader.Skip();
+                    ParseSxxExxSystem(reader, result);
                     break;
                 default: // Handles empty string and nulls
                     reader.Skip();
                     break;
             }
         }
+        public void ParseSxxExxSystem(XmlReader reader, XmlTvProgram result)
+        {
+            //// <episode-num system="SxxExx">S012E32</episode-num
+            string value = reader.ReadElementContentAsString();
+            Regex exp = new Regex("s([0-9]+)e([0-9]+)",RegexOptions.IgnoreCase);
+            Match res = exp.Match(value);
```

Comment:
can we combine two lines into
```suggestion
            Match res = new Regex("s([0-9]+)e([0-9]+)", RegexOptions.IgnoreCase).Match(value);
```
?